components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/src/main/java/org/wso2/carbon/identity/claim/metadata/mgt/ClaimMetadataManagementAdminService.java

```diff
@@ -162,6 +162,11 @@ public void addLocalClaim(LocalClaimDTO localClaim) throws ClaimMetadataExceptio
 
             IdentityClaimManagementServiceDataHolder.getInstance().getClaimManagementService()
                     .addLocalClaim(claim, tenantDomain);
+        } catch (ClaimMetadataClientException e) {
+            if (log.isDebugEnabled()) {
+                 log.debug(e.getMessage());
```

Comment:
e is missing

Reply:
Fix : https://github.com/wso2/carbon-identity-framework/pull/2926